### PR TITLE
Added links to error details object

### PR DESCRIPTION
**Pull request by ardiesaeidi**
Adds `@links` to `ErrorDetail` object. 

- Allows clients the ability to fetch data related to the attribute

### CODE REVIEW

File: versions/1.0/index.md [hunk 1]
```diff
@@ -426,36 +426,46 @@ An `Error` **MAY** have the following:
...
             "@type": "ErrorDetail",
-            "source": "/given_name",
-            "description": "Must not be empty"
+            "source": "/first_name",
+            "description": "Must not be empty and a mininum of 4 characters"
         },
         {
             "@type": "ErrorDetail",
-            "source": "/family_name",
+            "@link": {
```
Comment (iameugenejo) [inline versions/1.0/index.md]: Should it be `@link` or `@links` ? The description above says `@links` and the example here says `@link`

File: versions/1.0/index.md [hunk 2]
```diff
@@ -426,36 +426,46 @@ An `Error` **MAY** have the following:
...
+                "account": {
+                    "href": "/v1/account",
+                    "base_path": "https://api.xyz.co",
```
Comment (teehill) [inline versions/1.0/index.md:462]: This looks like a host not a base_path

File: versions/1.0/index.md (hunk 1 above)
Comment (thehelix112) [reply versions/1.0/index.md]: +1 should be links

File: versions/1.0/index.md (hunk 2 above)
Comment (thehelix112) [reply versions/1.0/index.md:462]: That matches the spec on the main hyperion page.

**Review by thehelix112 — CHANGES_REQUESTED** [review]
Need link changed to links and then good to go

**Review by thehelix112 — APPROVED** [review]
Lgtm

File: versions/1.0/index.md (hunk 2 above)
Comment (teehill) [reply versions/1.0/index.md:462]: @thehelix112 ... actually nvm base_path isn't used in the section I was looking at.

**Pull request merged**